src/pages/Detail/Detail.js

```diff
@@ -1,7 +1,84 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import DetailTopImg from './DetailTopImg';
+import DetailMain from './DetailMain/DetailMain';
+import DetailAside from './DetailAside/DetailAside';
+import styled from 'styled-components';
 
 const Detail = () => {
-  return <div>Detail</div>;
+  const [detailData, setData] = useState({});
+  const params = useParams();
+  const { detailId } = params;
+
+  useEffect(() => {
+    fetch(`http://10.58.3.52:8000/petsitters/detail/${detailId}`)
+      .then(response => response.json())
+      .then(data => setData(data.result));
+  }, []);
+
+  const plusCount = countDay => {
+    if (countDay !== 0) {
+      if (detailData.count === 6) {
+        alert('최대 6마리까지만 선택 가능합니다');
+      } else {
+        const newCount = { ...detailData };
+        newCount.count++;
+        setData(newCount);
+      }
+    } else {
+      alert('날짜를 먼저 선택해주세요');
+    }
+  };
+
+  const minusCount = () => {
+    if (detailData.count === 0) {
+      setData(detailData);
+    } else {
+      const newCount = { ...detailData };
+      newCount.count--;
+      setData(newCount);
+    }
+  };
+
+  return (
+    <>
+      {detailData.id && (
```

Comment:
return하는 JSX에서 조건부로 전체를 보여주는 것보다, 조건에 맞지 않을 때 early return하는 게 더 깔끔할 것 같습니다.
```suggestion
  if(!detailData.id) return <>데이터가 없습니다!</>
  
  return (
    <DetailWrapper>
```